MapboxMobileEvents/MMEEventsManager.m

```diff
@@ -72,7 +72,7 @@ - (void)initializeWithAccessToken:(NSString *)accessToken userAgentBase:(NSStrin
     
     [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(pauseOrResumeMetricsCollectionIfRequired) name:UIApplicationDidEnterBackgroundNotification object:nil];
     [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(pauseOrResumeMetricsCollectionIfRequired) name:UIApplicationDidBecomeActiveNotification object:nil];
-    if (&NSProcessInfoPowerStateDidChangeNotification != NULL) {
+    if (@available(iOS 9.0, *)) {
```

Comment:
Unfortunately, I don't think we can use the @available feature since that is only supported since Xcode 9. This project needs to support development with prior versions of Xcode where @available did not yet exist for Objective-C.

Reply:
Also, this check needs to be a runtime check to help with the case of running on an older version of iOS even if the library was compiled with a newer iOS SDK.